content/pubs/pubs.bib

```diff
@@ -1,3 +1,17 @@
+@article{sfn2018chapter,
+  title={Graph Classification Using Signal-Subgraphs: Applications in Statistical Connectomics},
+  author={Vogelstein, Joshua T},
```

Comment:
Looks like you are missing the other authors.

Reply:
Added, nice catch.